net/udp/udp_recvfrom.c

```diff
@@ -126,19 +126,19 @@ static void udp_recvpktinfo(FAR struct udp_recvfrom_s *pstate,
  *   Copy the read data from the packet
  *
  * Input Parameters:
- *   dev      The structure of the network driver that generated the event.
+ *   dev      The structure of the network driver that generated the event
  *   pstate   recvfrom state structure
  *
  * Returned Value:
- *   The number of bytes taken from the packet.
+ *   None.
  *
  * Assumptions:
  *   The network is locked.
  *
  ****************************************************************************/
 
-static size_t udp_recvfrom_newdata(FAR struct net_driver_s *dev,
-                                   FAR struct udp_recvfrom_s *pstate)
+static inline size_t udp_recvfrom_newdata(FAR struct net_driver_s *dev,
```

Comment:
maybe we can omit `inline`

Reply:
This is original style, there are similar inline functions in this file